react-jsx/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-jsx",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.16.0",
+    "@babel/plugin-transform-react-jsx": "^7.16.0",
+    "babel-loader": "^8.2.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "webpack": "^5.65.0",
+    "webpack-cli": "^4.9.1"
+  }
```

Comment:
Please make sure you're installing dev dependencies as `"devDependencies"`, not regular `"dependencies"`.

---

![image](https://user-images.githubusercontent.com/16143771/145635172-1c613198-062c-4f9b-89d3-d5ca7d163c10.png)